common/djangoapps/labster/course/views.py

```diff
@@ -306,9 +306,12 @@ def course_handler(request, course_key_string=None):
                 delete_course_and_groups(course_key, request.user.id)
                 return Response(status=status.HTTP_204_NO_CONTENT)
 
-    except LicenseStoreError:
-        raise HttpResponseBadRequest({
-            'ErrMsg': 'Incorrect license format.'
+    except LicenseStoreError as err:
+        log.exception(
+            'Incorrect licenses format: %s - %r', request.json.get('licenses', None), err
```

Comment:
we have a bit different messages in the log and resonse. So, I do not see any reason to use variable here.